src/layouts/center_main.rs

```diff
@@ -109,7 +109,7 @@ pub fn update(workspace: &Workspace, windows: &mut Vec<&mut Window>, tags: &mut
         if let Some(first) = iter.next() {
             first.set_height(workspace.height());
             first.set_width(primary_width);
-            first.set_x(workspace.x() + primary_x);
+            first.set_x(primary_x);
```

Comment:
This change actually fixes a pre-existing issue where `x` was calculated wrong if two windows were open and`workspace.x()` is not `0`.

In the previous version, `workspace.x()` was added to `primary_x`, but `primary_x`already contained the `workspace.x()` value when two windows were open, see line number 83.

I changed that so that `primary_x` always accounts for `workspace.x()`, so there is no need to add it again here.